RPIPs/RPIP-bounty_mechanism.md

```diff
@@ -0,0 +1,132 @@
+
+#### Payout
+* Bounty incentives SHOULD be paid out of the GMC budget.
+* Incentive payouts SHOULD take place within the GMC's regular payout schedule.
+* Incentive payouts MUST be denominated in dollars and SHOULD be paid out in one of the GMC's accepted payout token options, selected by the recipient.
+
+#### Lever A: Proposal Incentive
+The proposal incentive lever aims to influence the number of bounty proposals that are received by the GMC.
+* This incentive SHOULD be paid to the proposal author for a bounty proposal that is adopted by the GMC.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on proposal quality and completeness.
+* Bounty definitions do not need to be included for a proposer to receive this incentive.
+
+#### Lever B: Definition Incentive
+The definition incentive lever aims to influence the quality and completeness of bounty definitions accompanying bounty proposals.
+* This incentive SHOULD be paid to the definition author for bounty definitions provided alongside a bounty proposal that is adopted by the GMC.
+* This incentive payout SHOULD be paid a maximum of once per proposal, even if the bounty proposal is split into multiple definitions.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on definition quality and completeness.
+
+#### Lever C: Support Incentive
+The support incentive lever aims to influence the quality and consistency of bounty work submitted by bounty hunters. It aims to do this by rewarding contacts that support, oversee, and assist bounty hunters meet the requirements of open bounties.
+* This incentive SHOULD be split equally between bounty contacts that meet the following conditions:
+  1. The bounty MUST be completed to the satisfaction of the GMC.
+  2. The contact MUST be publicly listed in the bounty definition of the completed bounty.
+  3. The contact MUST NOT be affiliated with the completing bounty hunter.
+  4. The bounty hunter does not comment negatively on the contact when asked by the GMC.
+* Contacts SHOULD NOT assist the bounty hunter directly with the bounty work.
+* Contacts SHOULD assist bounty hunters via feedback, direction, and oversight upon request.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion for any reason. Reasoning must be publicly stated by the GMC.
+
+#### Initial Lever Recommendations
+These are non-binding recommendations to the GMC for initial incentive lever settings. They are included as a starting point for discussions and are not intended to limit GMC decision-making.
+* For the first few months, levers SHOULD be set to higher levels, with the goal of reaching a meaningful number of open bounties.
+* Lever A and B SHOULD NOT be zero unless unallocated GMC funds are low.
+* Lever B SHOULD be higher than Lever A to incentivize well-defined bounty generation.
```

Comment:
Isn't this definitionally the case? Given community members "may provide a bounty definition alongside their bounty proposal", wouldn't they then be receiving both A and B? Or do you mean even within that structure B should be more because B is more than half the work within A+B?

Reply:
The second, added some clarification.